python/cugraph-pyg/cugraph_pyg/examples/start_dask.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copyright (c) 2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+WORKER_RMM_POOL_SIZE=28G \
+CUDA_VISIBLE_DEVICES=2,3 \
```

Comment:
Seems like this could cause a problem on workstations with only 2 GPUs. Maybe a comment explaining the assumption about the system so someone else can understand how to change it if necessary?

Reply:
I'll change this back to "1,2", this was an inadvertent commit